src/quo2/components/messages/author/style.cljs

```diff
@@ -4,7 +4,6 @@
 (def container
   {:flex           1
    :flex-wrap      :wrap
-   :height         18
```

Comment:
why is this being deleted? surely it will have quite a knock on effect and perhaps better done in it's own p.r so it's easier for us / QA to check?

Although it seems `author` is only used in its corresponding quo2 preview screen. Did you check there that it looks okay?

Reply:
yes it looks ok :) with height its looks not ok